deployment/src/main/java/io/quarkus/jgit/deployment/devui/GiteaDevUIProcessor.java

```diff
@@ -0,0 +1,24 @@
+package io.quarkus.jgit.deployment.devui;
+
+import java.util.Optional;
+
+import io.quarkus.deployment.annotations.BuildProducer;
+import io.quarkus.deployment.annotations.BuildStep;
+import io.quarkus.devui.spi.page.CardPageBuildItem;
+import io.quarkus.devui.spi.page.Page;
+import io.quarkus.jgit.deployment.GiteaDevServiceInfoBuildItem;
+
+public class GiteaDevUIProcessor {
+
+    @BuildStep
+    void createCard(Optional<GiteaDevServiceInfoBuildItem> info, BuildProducer<CardPageBuildItem> cardPage) {
+        info.ifPresent(i -> {
```

Comment:
Why making it `Optional`? If the `GiteaDevServiceInfoBuildItem` isn't produced, it won't call this method, right?

Reply:
No, the method will be called with a 'null' value.